src/components/ReportActionItem/ReportActionItemImages.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import RenderHTML from '../RenderHTML';
+import stylePropTypes from '../../styles/stylePropTypes';
+import Text from '../Text';
+import Image from '../Image';
+
+const propTypes = {
+    /** array of image and thumbnail URIs */
+    images: PropTypes.arrayOf(
+        PropTypes.shape({
+            thumbnail: PropTypes.string,
+            image: PropTypes.string,
+        }),
+    ).isRequired,
+
+    /** optional: max number of images to show in the row if different than images length */
```

Comment:
We can remove optional from the comments. The lack of `isRequired` already indicates that it's optional.

```suggestion
    /** optional: max number of images to show in the row if different than images length */
```